src/main/java/io/nats/client/api/External.java

```diff
@@ -91,4 +101,48 @@ public int hashCode() {
         result = 31 * result + (deliver != null ? deliver.hashCode() : 0);
         return result;
     }
+
+    /**
+     * Creates a builder for an External object.
+     * @return the builder.
+     */
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    /**
+     * Placement can be created using a Builder.
+     */
+    public static class Builder {
+        private String api;
+        private String deliver;
+
+        /**
+         * Set the api string.
+         * @param api the api
+         * @return the builder
+         */
+        public Builder api(String api) {
+            this.api = api;
+            return this;
+        }
+
+        /**
+         * Set the deliver string.
+         * @param deliver the deliver
+         * @return the builder
+         */
+        public Builder deliver(String deliver) {
+            this.deliver = api;
```

Comment:
Shouldn't this be `this.deliver = deliver;`?
Btw, I think should be covered by unit tests to avoid such mistakes.

Reply:
fixed and added test